ios/RPCModule.swift

```diff
@@ -48,31 +48,45 @@ class RPCModule: NSObject {
       }
   }
 
-  func saveWalletFile(_ data: String) {
-      let paths = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)
-      if let documentsDirectory = paths.first {
-        let fileName = "\(documentsDirectory)/wallet.dat.txt"
-        do {
-            try data.write(toFile: fileName, atomically: true, encoding: .utf8)
-        } catch {
-            NSLog("Error save wallet \(error.localizedDescription)")
+  func saveWalletFile(_ base64EncodedString: String) {
```

Comment:
What's the difference between `saveWalletFile` and `saveWalletBackupFile`?

It seems like they're the same function but with a different `let fileName`, and some different error strings?

Reply:
When the App change to another wallet (tapping the menu option) automatically save the old wallet file in another `backup` file just in case the user wants to come back the prior wallet. If the user change to another wallet, have the possibility to tap in `restore last backup` to return the prior wallet. Because of this we have two different wallet files: normal & backup. This feature exists almost forever... in zingo-mobile.

Reply:
ok, I'm going to simplify the code en swift...

Reply:
I did.